packages/nextjs/app/page.tsx

```diff
@@ -16,7 +22,8 @@ const Home: NextPage = () => {
           <p className="text-center text-lg">Get started by taking a look at your batch GitHub repository.</p>
           <p className="text-lg flex gap-2 justify-center">
             <span className="font-bold">Checked in builders count:</span>
-            <span>To Be Implemented</span>
+            <span className={`${!checkedInCounter.isFetched && "hidden"}`}>{Number(checkedInCounter.data)}</span>
```

Comment:
On initial render `checkedInCounter.data` is `undefined` and `Number(checkedInCounter.data)` give NaN , and we get this error in console: 

![Screenshot 2024-06-06 at 9 38 20 AM](https://github.com/BuidlGuidl/batch6.buidlguidl.com/assets/80153681/8e666a04-2b8e-4915-bdcd-6fa7c4e6ec11)

Instead of casting to Number you could do `checkedInCounter?.data?.toString()` since `checkedInCounter` is bigInt



---

Also btw, we shouldn't use `&&` in `className` and instead use ternary operator with empty string  `''`   

If you use `&&` if the condition is `false` it will use it as className: 

![Screenshot 2024-06-06 at 9 38 08 AM](https://github.com/BuidlGuidl/batch6.buidlguidl.com/assets/80153681/3632dba8-17d3-4df4-a927-7fd44ba11f25)